t/003-spider.t

```diff
@@ -12,10 +14,8 @@ SKIP: {
         skip "set TEST_SPIDER env var to test the spider", 2;
     }
 
-    eval "use Dezi::Aggregator::Spider";
-    if ( $@ && $@ =~ m/([\w:]+)/ ) {
-        skip "$1 required for spider test: $@", 3;
-    }
+    try_load_class("Dezi::Aggregator::Spider")
```

Comment:
This was actually to figure out what the missing dependency was - something that Dezi::Aggregator::Spider was "use"-ing. But this works too.

Reply:
Hrm, that's what I thought, but I tried uninstalling a module used by D::A::S to see if I could trigger the regexp part required for the skip, but the test would die with a compile time error and wasn't simply skipping the tests, so the eval didn't seem to be doing what it was intended to do.  Thanks for clearing that one up for me :-)